message MSG_DC_DM_TOKEN_IDENTIFY
{
    optional string encryptedToken = 1;
}

message MSG_DM_DC_TOKEN_IDENTIFY_ACK
{
    optional int32 statusCode = 1;
    optional string rawToken = 2;
}